docs/freqai-developers.md

```diff
@@ -0,0 +1,28 @@
+# Development 
+
+The class structure and details algorithmic overview is depicted in the following diagram: 
+
+![image](assets/freqai_algorithm-diagram.jpg)
+
+As shown, there are three distinct objects comprising `FreqAI`:
+
+* IFreqaiModel
+  * Singular persistent object containing all the necessary logic to collect data, store data, process data, engineer features, run training, and inference models. 
+* FreqaiDataKitchen
+  * A non-persistent object which is created uniquely for each unique asset/model. Beyond metadata, it also contains a variety of data processing tools. 
+* FreqaiDataDrawer
+  * Singular persistent object containing all the historical predictions, models, and save/load methods. 
+
+There are a variety of built-in prediction models which inherit directly from `IFreqaiModel` including:
+
+* CatboostRegressor
+* CatboostRegressorMultiTarget
+* CatboostClassifier
+* LightGBMRegressor
+* LightGBMRegressorMultiTarget
+* LightGBMClassifier
+* XGBoostRegressor
+* XGBoostRegressorMultiTarget
+* XGBoostClassifier
```

Comment:
Ain't having a list here dangerous - at least to some extend?

assuming we add another model - it's very likely that this will be forgotten.

The list should be there for sure - but directed at users "these are your options".
developer docs should focus on what's important for the developer - this list is not really - at least not in detail.

Reply:
@th0rntwig fixed this